src/SellerLabs/Research/Entities/SearchProduct.php

```diff
@@ -216,4 +221,28 @@ public function setWeight($weight)
     {
         $this->weight = $weight;
     }
+
+    /**
+     * @param array $rankings
+     */
+    public function setRankings(array $rankings)
+    {
+        $this->rankings = $rankings;
+    }
+
+    /**
+     * @param $relationships
+     */
+    public function setRelationships($relationships)
```

Comment:
setRelationship(RelationshipBag $relationship)  seing as getRelationships is returning that type